docs/user-guide/examples/README.txt

```diff
@@ -0,0 +1,6 @@
+.. _tutorials_and_examples_gallery:
+
+Tutorials and Examples gallery
+==============================
```

Comment:
Minor style issue: this headline  is at the section level, so the ones below it (each example/tutorial) should be a level deeper (so subsection)

For RST:
```
# with overline, for parts
* with overline, for chapters
= for sections
- for subsections
^ for subsubsections
" for paragraphs
```

Reply:
That would fix the wrong hierarchy here:
<img width="262" alt="image" src="https://github.com/cta-observatory/ctapipe/assets/11677812/659dea6f-405d-46fe-a860-93fd552c63f1">

Reply:
Why is this a `README.txt`  file and not a `.rst` file called index?

Reply:
It's just the default sphinx gallery style
>By default, Sphinx-Gallery creates all the files that are written in the sphinx-build directory, either by generating rst and images from a *.py in the gallery-source, or from creating index.rst from README.txt in the gallery-source.

I think I can change it
>You may also want to pass raw rst from the gallery-source to the sphinx-build[...] To accommodate this, you may set copyfile_regex in sphinx_gallery_conf. The following copies across rst files.
```python
sphinx_gallery_conf = {
    ...
   'copyfile_regex': r'.*\.rst',
}
```

Reply:
Maybe the copyfile_regex is not needed since source and build dir are the same in this case

Reply:
But if I do this the toctree will not be generated by sphinx-gallery

Reply:
> By default, Sphinx-Gallery creates all the files that are written in the sphinx-build directory, 

Why did you change that so that the generated files clutter the docs *source* directory?